lib/encode.ex

```diff
@@ -204,6 +205,46 @@ defmodule Jason.Encode do
     end
   end
 
+  defp map_camel_case(value, escape, encode_map) do
+    case Map.to_list(value) do
+      [] -> "{}"
+      [{key, value} | tail] ->
+        key = camelize(key)
```

Comment:
I'm sure that I saw this same code somewhere below at line #224. Can't we make it somehow not to have duplicate code?